### DISCUSSION TITLE
Extended ASCII characters displaying '?'

### DISCUSSION THREAD

**Discussion by Overdrive77** [CosmosOS/Cosmos#1698] [Q&A]
I'm using Console.Write() to draw a DOS styled box using ASCII box shape codes. But the output is displaying question marks instead of box shapes.

Create a new CosmosOS project and inside Run() method enter the following:

Console.WriteLine([█████]);
Console.WriteLine($"{new string((char)219, 5)}");

Would output:
[?????]
[?????]

Is this a bug? Or am I missing something? Does someone know how to include these chars?

For more information about the characters I'm interested in see: http://www.asciitable.com/ chars 176 - 223

**Comment by quajak** [comment]
The standard font does not have any symbol for that character. It is possible to set the font to a custom one which has a symbol for all extended ascii chars.

**Reply by lucasneessen** [reply]
Use..
Encoding.RegisterProvider(CosmosEncodingProvider.Instance);

Console.InputEncoding = Encoding.GetEncoding(437);
Console.OutputEncoding = Encoding.GetEncoding(437);
